### DISCUSSION TITLE
<title>Jest unexpected token error with React-Native tsx

### DISCUSSION THREAD

**Discussion by jsee53** [community/community#73802] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

1

I attempted to run Jest tests for TypeScript code in a React Native project. I expected the tests to run without errors. However, I encountered the

Jest encountered an unexpected token error

when using the statement 
import { createNativeStackNavigator } from '@react-navigation/native-stack';. 
The error occurred during the test execution, and I'm seeking a solution to resolve it and make the tests run as expected. How can I resolve this issue and run Jest tests successfully?

**Comment by mutasim77** [comment]
did you check it? [link](https://stackoverflow.com/questions/52188246/jest-encountered-an-unexpected-token-with-react-native#:~:text=Jest%20encountered%20an%20unexpected%20token%20This%20usually%20means%20that%20you,files%2C%20ignoring%20%22node_modules%22.)
